www/.eslintrc.json

```diff
@@ -1,3 +1,28 @@
 {
-  "extends": "next/core-web-vitals"
+  "parser": "@typescript-eslint/parser",
```

Comment:
gotta figure out how to run this in CI

Reply:
Tru dat. Filed https://github.com/zsol/py.wtf/issues/21